docs/advanced_usage.rst

```diff
@@ -289,4 +290,69 @@ Command Line Options
 
 .. cc-command-line-options::
 
+Choice Variables (1.1+)
+-----------------------
+
+Choice variables provide different choices when creating a project. Depending on an user's choice
+the template renders things differently.
+
+Basic Usage
+~~~~~~~~~~~
+
+For example, if you provide the follwing choice variable in your ``cookiecutter.json``::
+
+   {
+       "license": ["MIT", "BSD-3", "GNU GPL v3.0", "Apache Software License 2.0"]
+   }
+
+you'd get the following choices when running Cookiecutter::
+
+   Select license:
+   1 - MIT
+   2 - BSD-3
+   3 - GNU GPL v3.0
+   4 - Apache Software License 2.0
+   Choose from 1, 2, 3, 4 [1]:		
+
+Depending on an user's choice, a different license is rendered by Cookiecutter. 
+
+The above ``license`` choice variable creates ``cookiecutter.license``, which
+can be used like this::
```

Comment:
Please also mentioned that a choice variable is still a regular cookiecutter variable and can also be used as follows:

```
License
-------

Distributed under the terms of the `{{cookiecutter.license}}`_ license,
"pytest-{{cookiecutter.plugin_name}}" is free and open source software
```